Production/crab/configs/2018/aux/ZeroBias.txt

```diff
@@ -0,0 +1,6 @@
+lumiMask=/afs/cern.ch/cms/CAF/CMSCOMM/COMM_DQM/certification/Collisions18/13TeV/ReReco/Cert_314472-325175_13TeV_17SeptEarlyReReco2018ABC_PromptEraD_Collisions18_JSON.txt
+sampleType=Run2018ABC
+
+ZeroBias_Run2018A /ZeroBias/Run2018A-17Sep2018-v1/MINIAOD
```

Comment:
- move from aux folder to the "main" folder together with other txt
- add config for ZeroBias_Run2018D
- add `requireGenMatch=False`

Reply:
Just small question for the Run2018D there are following sets:
/ZeroBias/Run2018D-12Nov2019_UL2018_rsb-v1/MINIAOD
/ZeroBias/Run2018D-ForValUL2018-v1/MINIAOD
/ZeroBias/Run2018D-PromptReco-v1/MINIAOD
/ZeroBias/Run2018D-PromptReco-v2/MINIAOD

1) which one is to add?
2) the last run in this datasets has number 325175, so the luminosity mask seems to cover it, but I don't understand why in the name of lumiMask it is written ReReco2018ABC_PromptEraD... can it be applied to Run2018D?

Reply:
Before UltraLegacy (UL) campaign, for 2018D only a prompt reconstruction is available, while for 2018ABC re-reco datasets were produced. You should use `/ZeroBias/Run2018D-PromptReco-v2/MINIAOD` with the same lumi mask file as for 2018ABC.